scripts/servers-build.sh

```diff
@@ -7,7 +7,7 @@ echo "WORKSPACE=$WORKSPACE"
 
 build_servers_repositories() {
     # Use release tool to query for available docker names.
-    for docker in $($WORKSPACE/integration/extra/release_tool.py --list docker ); do (
+    for docker in $($WORKSPACE/integration/extra/release_tool.py --list docker | egrep -v 'mender-client-.*|mender-monitor-qemu-commercial'); do (
```

Comment:
Now that the mapping produces multiple git repos for the docker client images, it's easier to skip this all together, than add extra logic imo.

Reply:
I would perhaps suggest that you still use the `case` statement to do the matching, since it also supports `|`, and is a bit more visible, but otherwise I agree with the general approach.

Reply:
Alright.
Added the case statements back, and moved the subshell invocation, due to the extra embedded for loop.

Note now that it will loop through the `mender` and `mender-monitor` repos mutliple times, due to the N:N mappings, but they're all no-op's so no matter.

Might be something to keep in mind for when we get N:N mappings for backend components though, like possible we can through the `mender-gateway` ?
@kacf